filecheck/checker.ts

```diff
@@ -269,29 +269,60 @@ function canCheckFile(filePath: string) {
 async function resolveDirectory(file: string): Promise<string[]> {
   const stats = await fs.lstat(file);
   if (stats.isDirectory()) {
-    const api = new fdir()
-      .withErrors()
-      .withFullPaths()
-      .filter((filePath) => canCheckFile(filePath))
-      .crawl(file);
+    const api = new fdir().withErrors().withFullPaths().crawl(file);
     return api.withPromise() as Promise<PathsOutput>;
-  } else if (stats.isFile() && canCheckFile(file)) {
+  } else if (stats.isFile()) {
```

Comment:
... and then we would also need to check for `isRelevantFile(file)` here.

Reply:
done